### PR TITLE
logstash: fix unlinked logstash plugin command

### PR DESCRIPTION
**Pull request by mattnworb**
Logstash also provides a `plugin` command/script for managing plugins.
The current recipe does not expose this command, forcing the user to
look through the Cellar to find the correct path.

### CODE REVIEW

**Comment by DomT4** [comment]
`plugin` seems awfully generic for a globally-available executable name, which somewhat concerns me. I'll let some of the other maintainers agree/disagree here though.

**Comment by mattnworb** [comment]
I agree, but I wasn't sure if it was conventional or not for homebrew recipes to rename commands. `logstash-plugin` seems like a sensible choice if renaming things doesn't go against conventions.

**Comment by DomT4** [comment]
Where the name is generic it's not too uncommon. You can do something like:

```ruby
bin.install_symlink libexec/"bin/plugin" => "logstash-plugin"
```

**Comment by mikemcquaid** [comment]
> Where the name is generic it's not too uncommon.

:+1: to `logstash-plugin`

**Comment by mattnworb** [comment]
I added a commit to change the name to `logstash-plugin` as suggested and added a caveat to mention the name change. Thanks for the suggestion!

File: Library/Formula/logstash.rb [hunk 1]
```diff
@@ -24,14 +24,17 @@ def install
...
     inreplace %w[bin/logstash.lib.sh], /^LOGSTASH_HOME=.*$/, "LOGSTASH_HOME=#{libexec}"
     libexec.install Dir["*"]
     bin.install_symlink libexec/"bin/logstash"
+    bin.install_symlink libexec/"bin/plugin" => "logstash-plugin"
   end
 
   def caveats; <<-EOS.undent
     Please read the getting started guide located at:
       https://www.elastic.co/guide/en/logstash/current/getting-started-with-logstash.html
+    The logstash `plugin` command is available under `logstash-plugin`.
```
Comment (mikemcquaid) [inline Library/Formula/logstash.rb]: `under the name`?

Comment (mattnworb) [inline Library/Formula/logstash.rb]: I reworded this to sound less weird.

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew/commit/9107a1e033f5a9aab801bfbfed13b5c4d6f052d8. Thank you for your contribution to Homebrew @mattnworb; we appreciate it! :smiley_cat:

**Pull request closed without merging**